src/mysteryshack/web/app.rs

```diff
@@ -483,20 +483,21 @@ fn webfinger_response(request: &mut Request) -> IronResult<Response> {
                         // Spec version
                         "http://remotestorage.io/spec/version" => version,
 
-                        // OAuth
+                        // OAuth as in draft-06
                         "http://tools.ietf.org/html/rfc6749#section-4.2" => oauth_url.serialize(),
 
-                        // Support for providing the access token via URL query param
-                        "http://tools.ietf.org/html/rfc6750#section-2.3" => false,
+                        // No support for providing the access token via URL query param as in
+                        // draft-06
+                        "http://tools.ietf.org/html/rfc6750#section-2.3" => None,
 
-                        // Content-Range as in draft-02
-                        "http://tools.ietf.org/html/rfc2616#section-14.16" => false,
+                        // No Content-Range as in draft-02
```

Comment:
No, because it is only mentioned in draft-02 and missing from newer versions. This is for older remoteStorage.js versions (see also README) and shouldn't conflict with newer clients.